### PR TITLE
upgrade stylelint

### PR DESCRIPTION
**Pull request by akdetrick**
## Description

**stylelint + cleanup**

- Updated `stylelint` to make ignore comments less flakey
- Re-ran lint and fixed all errors in CSS
- Fixed warning in `menu` test

### CODE REVIEW

**Comment by akdetrick** [comment]
holding for more testing/fixes

**Comment by akdetrick** [comment]
should be good to go now; removing label

File: src/components/popovers/menu.test.js [hunk 1]
```diff
@@ -12,20 +18,20 @@ describe('Popover component', () => {
...
   it('matches snapshot (JSX items)', () => {
     const wrapper = mount(
-      <Menu trigger={<button>trigger</button>}>
-        <MenuItem><p>First</p></MenuItem>
-        <MenuItem><p>Second</p></MenuItem>
-        <MenuItem><p>Third</p></MenuItem>
+      <Menu trigger={<a href="#">trigger</a>}>
+        <MenuItem onSelect={() => {}}><p>First</p></MenuItem>
+        <MenuItem onSelect={() => {}}><p>Second</p></MenuItem>
+        <MenuItem onSelect={() => {}}><p>Third</p></MenuItem>
```
Comment (rey-wright) [inline src/components/popovers/menu.test.js]: Why no button here anymore?

Comment (rey-wright) [reply src/components/popovers/menu.test.js]: also, we could utilize storybook actions here instead of noop potentially?

Comment (akdetrick) [reply src/components/popovers/menu.test.js]: native `button` inside of a button is no bueno - avoids a warning

Comment (akdetrick) [reply src/components/popovers/menu.test.js]: agree about using actions - Filed a follow up, UIP-626

**Pull request merged**